frame/election-provider-multi-phase/src/weights.rs

```diff
@@ -57,101 +57,105 @@ pub trait WeightInfo {
 pub struct SubstrateWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	fn on_initialize_nothing() -> Weight {
-		(22_730_000 as Weight)
-			.saturating_add(T::DbWeight::get().reads(7 as Weight))
+		(24_579_000 as Weight)
+			.saturating_add(T::DbWeight::get().reads(8 as Weight))
 	}
 	fn on_initialize_open_signed() -> Weight {
-		(112_051_000 as Weight)
-			.saturating_add(T::DbWeight::get().reads(8 as Weight))
+		(87_463_000 as Weight)
+			.saturating_add(T::DbWeight::get().reads(10 as Weight))
 			.saturating_add(T::DbWeight::get().writes(4 as Weight))
 	}
 	fn on_initialize_open_unsigned_with_snapshot() -> Weight {
-		(112_165_000 as Weight)
-			.saturating_add(T::DbWeight::get().reads(8 as Weight))
+		(87_381_000 as Weight)
+			.saturating_add(T::DbWeight::get().reads(10 as Weight))
 			.saturating_add(T::DbWeight::get().writes(4 as Weight))
 	}
 	fn on_initialize_open_unsigned_without_snapshot() -> Weight {
-		(21_039_000 as Weight)
+		(18_489_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(1 as Weight))
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
 	}
 	fn elect_queued() -> Weight {
-		(7_362_949_000 as Weight)
+		(6_038_989_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(2 as Weight))
 			.saturating_add(T::DbWeight::get().writes(6 as Weight))
 	}
-	fn submit_unsigned(v: u32, _t: u32, a: u32, d: u32, ) -> Weight {
+	fn submit_unsigned(v: u32, t: u32, a: u32, d: u32, ) -> Weight {
 		(0 as Weight)
-			// Standard Error: 21_000
-			.saturating_add((3_933_000 as Weight).saturating_mul(v as Weight))
-			// Standard Error: 21_000
-			.saturating_add((13_520_000 as Weight).saturating_mul(a as Weight))
-			// Standard Error: 107_000
-			.saturating_add((2_880_000 as Weight).saturating_mul(d as Weight))
-			.saturating_add(T::DbWeight::get().reads(6 as Weight))
+			// Standard Error: 12_000
+			.saturating_add((3_480_000 as Weight).saturating_mul(v as Weight))
+			// Standard Error: 42_000
+			.saturating_add((194_000 as Weight).saturating_mul(t as Weight))
+			// Standard Error: 12_000
+			.saturating_add((10_498_000 as Weight).saturating_mul(a as Weight))
+			// Standard Error: 63_000
+			.saturating_add((3_074_000 as Weight).saturating_mul(d as Weight))
+			.saturating_add(T::DbWeight::get().reads(7 as Weight))
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
```

Comment:
Weights were not updated when making these changes?

cc @coriolinus @kianenigma